python_ta/contracts/__init__.py

```diff
@@ -5,18 +5,28 @@
 import wrapt
 
 
-def check_all_contracts(module_name: str = '__main__') -> None:
+def check_all_contracts(*args, **kwargs) -> None:
```

Comment:
You can use a keyword-only argument (https://www.python.org/dev/peps/pep-3102/) instead of `**kwargs`. This is a tighter specification for this function.